client/src/components/LayoutComponents/Home/style.js

```diff
@@ -0,0 +1,9 @@
+const Style = {
+	paper: {
+		background: '#E4DDDD',
+		width: 1170,
```

Comment:
The style not responsive and it's large to screen in desktop one!

Reply:
really on mine its not but you are right about responsive I will do it today